### PR TITLE
UAG-168 | React Admin UI

### PR DESCRIPTION
**Pull request by supriyajjadhav**
### Description
Updated UAG dashboard Latest design in react

### Types of changes
Breaking change

### Checklist:
- [x] My code is tested
- [x] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: admin-core/admin-loader.php [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+ * Class Admin_Loader.
+ */
+class Admin_Loader {
+
+	/**
+	 * Instance
+	 *
+	 * @access private
+	 * @var object Class object.
+	 * @since 1.0.0
```
Comment (RajkiranBagal) [inline admin-core/admin-loader.php:28]: Update the Since tag to x.x.x everywhere

File: admin-core/assets/src/sass/colors/_colors.scss [hunk 2]
```diff
@@ -0,0 +1,26 @@
+$color-primary: #f06335;
+
+/*
```
Comment (RajkiranBagal) [inline admin-core/assets/src/sass/colors/_colors.scss:3]: We can remove unwanted code.

File: admin-core/assets/src/sass/colors/_structure.scss [hunk 3]
```diff
@@ -0,0 +1,22 @@
+/*----------  Site Basic Structure  ----------*/
+
+$width-site-main: 					100%;
+$width-site-content: 				960px;
```
Comment (RajkiranBagal) [inline admin-core/assets/src/sass/colors/_structure.scss:4]: Are we using these variables? If not we can remove them

File: admin-core/assets/src/sass/colors/_typography.scss [hunk 4]
```diff
@@ -0,0 +1,21 @@
+// Typography
+
+/*----------  Font Size  ----------*/
+$font-size-root:     		14px !default; // Applied to <html> element only
+$font-size-base:     		1 !default;
+$font-size-h1:       		2em !default;
```
Comment (RajkiranBagal) [inline admin-core/assets/src/sass/colors/_typography.scss:6]: Are we using these variables? If not we can remove them

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

**Review by sandeshjangam — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**